replication/const.go

```diff
@@ -197,6 +202,12 @@ func (e EventType) String() string {
 		return "TransactionPayloadEvent"
 	case HEARTBEAT_LOG_EVENT_V2:
 		return "HeartbeatLogEventV2"
+	case MARIADB_WRITE_ROWS_COMPRESSED_EVENT_V1:
+		return "MariadbWriteRowsCompressedEventV1"
+	case MARIADB_UPDATE_ROWS_COMPRESSED_EVENT_V1:
+		return "MariadbUpdateRowsCompressedEventV1"
+	case MARIADB_DELETE_ROWS_COMPRESSED_EVENT_V1:
+		return "MariadbDeleteRowsCompressedEventV1"
```

Comment:
we can also support MARIADB_START_ENCRYPTION_EVENT and MARIADB_QUERY_COMPRESSED_EVENT

Reply:
Added `MARIADB_QUERY_COMPRESSED_EVENT`.
Moved the uncompression functions to `mysql` package as it is shared.
<img width="505" alt="Screenshot 2023-05-05 at 20 56 58" src="https://user-images.githubusercontent.com/232147/236538791-27738113-5b0d-4ff3-84d2-9baf5cca93f5.png">
<img width="313" alt="Screenshot 2023-05-05 at 20 59 41" src="https://user-images.githubusercontent.com/232147/236538803-eea66099-2dfd-4522-b1e3-9ae1d11d0cdc.png">


As for `MARIADB_START_ENCRYPTION_EVENT`  - I added this event only to fill enum values. It is tricky to support it. I added the text description, but since everything (including next event pos) is encrypted this library crashes when it is enabled.
<img width="1208" alt="Screenshot 2023-05-05 at 21 21 57" src="https://user-images.githubusercontent.com/232147/236538773-5e5486c2-2373-491c-abfb-4a2ec7f47478.png">